server/gameconnection.py

```diff
@@ -417,6 +349,21 @@ def send_message(self, message):
                 (teamkiller_id, victim_id, self.game.id, gametime)
             )
 
+    async def handle_ice_message(self, receiver_id, ice_msg):
+        peer = self.player_service.get_player(receiver_id)
+        if not peer:
+           self._logger.info("Ignoring ICE message for unknown player: %s", receiver_id)
+           return
+
+        game_connection = peer.game_connection
+        if not game_connection:
+            self._logger.info("Ignoring ICE message for player without game connection: %s", receiver_id)
+            return
+
+        game_connection.send_message(dict(command="IceMessage", args=[int(self.player.id), ice_msg]))
```

Comment:
Use `{}` syntax. Why are we calling `int` on `player.id` isn't that an int already?